### DISCUSSION TITLE
Suppress built in volume OSD

### DISCUSSION THREAD

**Discussion by gaevoc** [MonitorControl/MonitorControl#1308] [Q&A] [answered]
Hi,
thank you for this great piece of software.
I have an LG 27UK850 which works like a breeze, just wonder if there is any way to suppress the volume built in OSD and display only the MacOS one. I am on Ventura currently.

Thank you!
Gaetano

**Comment by waydabber** [comment, marked as answer]
Unfortunately not with LGs. By sending an other (even invalid) command the OSD is taken away but this will create an annoying flickering effect on these displays.

**Reply by gaevoc** [reply]
I understand. 
No problem at all, I will suppress the MacOS volume OSD just to avoid the overlapping
